### PR TITLE
Infection Gamemode

### PR DESCRIPTION
**Pull request by Bibby0110**
### Intent of your Pull Request
Adds a 7 upgrade tree to the Blob

#### Changelog

:cl:  
rscadd: YEETED Blob v2 into existence
rscdel: YEETED blob
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
51 commits

**Comment by Darkstick** [comment]
What the heck is this, put a description on it you butt

**Comment by Bibby0110** [comment]
no

**Comment by Bibby0110** [comment]
merge it and find out

File: code/modules/antagonists/blob/infection_admin.dm [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+		return
+
+	for(var/D in GLOB.crewDatum)
+		if(istype(D, /datum/infection_crew))
+			var/datum/infection_crew/crew = D
+
+			crew.changeDefcon(2)
+
+/client/proc/start_blob()
+	set name = "Start Blob Victory Timer"
```
Comment (Hopekz) [inline code/modules/antagonists/blob/infection_admin.dm:100]:
```suggestion
	set name = "Make the Blob dab on everyone"
```

File: code/modules/antagonists/blob/infection_admin.dm [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+		if(istype(D, /datum/infection_crew))
+			var/datum/infection_crew/crew = D
+			var/which = input(usr, "Which missile?", "Missile") as null|anything in list("AGM", "Fatty", "Sidewinder", "Cancel")
+			if(which == "Cancel")
+				return
+			var/amount = input(usr, "Award how many?") as num
+			crew.addBoom(which, amount)
+
+/client/proc/giveFactoryUpgrade()
+	set name = "Give Factory Upgrade Points"
```
Comment (Hopekz) [inline code/modules/antagonists/blob/infection_admin.dm:179]:
```suggestion
	set name = "Give Epic Gamer Upgrade Points"
```

**Comment by ShadowDeath6** [comment]
interesting pull request

**Review by ThatLing — CHANGES_REQUESTED** [review]
Add a description and changelog or I will close this

**Comment by Whoneedspacee** [comment]
based

**Pull request closed without merging**